SwiftStats/stats.swift

```diff
@@ -437,8 +437,8 @@ public struct Common {
 			let num = (((a[3]*z + a[2])*z + a[1])*z) + a[0]
 			let den = ((((b[3]*z + b[2])*z + b[1])*z + b[0])*z + 1.0)
 			var x = y*num/den
-			x = x - (erf(x) - y)/(2.0/sqrt(M_PI)*exp(-x*x))
-			x = x - (erf(x) - y)/(2.0/sqrt(M_PI)*exp(-x*x))
+			x = x - (erf(x) - y)/(2.0/sqrt(Double.pi)*exp(-x*x))
```

Comment:
is there any reason we aren't doing this instead? Seems I defined the `pi` variable but don't use it here

```suggestion
			x = x - (erf(x) - y)/(2.0/sqrt(pi)*exp(-x*x))
```

Reply:
I did see that you had defined a constant.  I went with the "make the fewest changes" option.  Some people really don't like it when others change their code--looks like you're happy to see contributions, which is great :o)

Reply:
I have fixed this in a recent commit.  I'm just waiting on you to tell me how you'd like pull requests.